src/modules/esl-tooltip/core/esl-tooltip.ts

```diff
@@ -41,8 +41,11 @@ export class ESLTooltip extends ESLPopup {
     return els.length ? els[els.length - 1] : null;
   }
 
+  public static isOpen(): boolean {
+    return ESLTooltip.sharedInstance.open;
+  }
```

Comment:
In case we gonna use in such API representation - please use getter accessor to be consistent with API of other components
```suggestion
  public static get open(): boolean {
    return ESLTooltip.sharedInstance.open;
  }
```

Reply:
Thanks! Fixed